### PR TITLE
August 9 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 494416a |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-495.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by georgiana-caras — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20240809_amplify.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+| AWS Gateway using SDK 2.0                  | DA=v1.2.0 / TA-v1.2.0    | DA=v1.1.64 / TA-v1.1.67          |
+| Azure latest release                       | TBD / TBD                | DA=v1.1.70 / TA=v1.1.76          |
+| Istio 1.9.5                                | TBD / TBD                | DA=v1.0.36 / TA=v2.0.82          |
+| Apigee Edge                                | 0.2.0                    | 0.1.38                           |
+| Mulesoft Anypoint platform v3              | 1.2.0                    | 1.1.48                           |
+| Software AG webMethods                     | 0.0.29                   | 0.0.25                           |
+| Kong Gateway                               | 1.1.0                    | 1.0.9                            |
+| GitLab                                     | 1.1.0                    | 1.0.6                            |
+| Kafka Cluster                              | DA=v1.1.0 / TA=v1.1.0    | DA=v1.0.6 / TA=v1.0.6            |
+| IBM API Connect Gateway                    | DA=v1.1.0 / TA=v1.1.0    | DA=v1.0.0 / TA=v1.0.0            |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240809_amplify.md:31]: I think the minimum version values should also be on agent_agentsdk_support_policy.md

Comment (jcollins-axway) [reply content/en/docs/amplify_relnotes/20240809_amplify.md:31]: ...or in the _index.md file

Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240809_amplify.md:31]: will update those as well in a separate PR

**Review by robertengaxway — APPROVED** [review]
approved

**Review by jcollins-axway — APPROVED** [review]

**Pull request merged**